src/com/UsbDevice.cpp

```diff
@@ -120,11 +123,18 @@ namespace plug::com::usb
         std::vector<std::uint8_t> buffer(dataSize);
         int transfered{0};
 
-        if (const auto result = libusb_interrupt_transfer(handle_.get(), endpoint, buffer.data(), dataSize, &transfered, usbTimeout.count()); (result != LIBUSB_SUCCESS) && (result != LIBUSB_ERROR_TIMEOUT))
+        const auto result = libusb_interrupt_transfer(handle_.get(), endpoint, buffer.data(), dataSize, &transfered, usbTimeout.count());
+#ifndef NDEBUG
```

Comment:
I added this output so I could tell when the  libusb call was succeeding and when it was timing out